dataland-backend/src/main/kotlin/org/dataland/datalandbackend/api/CompanyAPI.kt

```diff
@@ -43,9 +43,10 @@ interface CompanyAPI {
         ResponseEntity<StoredCompany>
 
     @Operation(
-        summary = "Retrieve specific companies by name or just all companies from the data store.",
-        description = "Companies identified via the provided company name are retrieved. " +
-            "If company name is an empty string, all companies in the data store are returned."
+        summary = "Retrieve specific companies by name/identifier/index or just all companies from the data store.",
+        description = "Companies identified via the provided company name/identifier/index are retrieved. " +
+            "If an empty string is passed as search argument, all companies in the data store will be returned." +
+            "If selectedIndex is not null, all companies with the given stock index will be returned."
```

Comment:
Besseres Wording:  ...all companies listed in the given stock index ...

Reply:
Ich habe das so umformuliert das es klar wird das hier alle Firmen in Dataland zurückkommen die zu dem Index gehören (bei deiner Formulierung wurde man erwarten es kommen alle DAX Firmen aber wenn Dataland nur 29/40 DAX Firmen kennt würden ja nur 29 kommen)